packages/imperative/src/events/src/doc/IEventJson.ts

```diff
@@ -9,36 +9,37 @@
 *
 */
 
+import { EventTypes } from "../EventConstants";
+import * as fs from "fs";
+
 /**
- * Imperative Event JSON representation
+ * Imperative Event JSON representation for user interpretation
  * @export
- * @interface IImperativeEventJson
+ * @interface IEventJson
  */
-export interface IImperativeEventJson {
+export interface IEventJson {
     /**
      * The time in which the event occurred
      */
-    time: string;
+    eventTime: string;
     /**
-     * The type of event that occurred
+     * The name of event that occurred
      */
-    type: string;
+    eventName: string;
     /**
-     * The application name that triggered the event
+     * The type of event that occurred
      */
-    source: string;
+    eventType: EventTypes;
     /**
-     * The ID of the event that occurred
+     * The application name that triggered the event
      */
-    id?: string;
+    appName: string;
     /**
-     * The location in which the event was emitted (User vs Shared)
+     * The file path for information on the emitted event
      */
-    loc?: string;
+    eventFilePath: string;
     /**
-     * The indicator of user-specific (if true) or shared (if false) events
+     * List of watchers to eventually close
      */
-    user?: boolean;
+    subscriptions: fs.FSWatcher[];
```

Comment:
Is the `IEventJson` interface intended to be JSON serializable? If so perhaps the `subscriptions` object should be stored somewhere else since I don't think the `fs.FSWatcher` type can be represented as JSON

Reply:
Thanks for pointing it out. The answer is "kind of" 😋 
For serializing the object we created a `toJson` method as part of the Event class 😋 
We made subscription optional and properly typed the return value of said function 🥳